### ISSUE TITLE
Structure panel missing top level functions defined after nested let foo = function

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#175]
plugin version: 0.78.2

### Description

Structure panel should show `cvt_gram_sym_list` and `cvt_gram_prod`.

![image](https://user-images.githubusercontent.com/1253341/60941570-772fed80-a294-11e9-9fdb-519d1a47113c.png)

```
let cvt_ext prod =
 let from_ext = function
    | _ -> ""
 in
 ()

and cvt_gram_sym_list l =
  let get_sym = function
    | _ -> ""
  in
  ()

and cvt_gram_prod p = ()
```

**Comment by giraud** [comment]
fixed in 0.82

**Comment by jfehrle** [comment]
Looks good.
